File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/release/GraduationDesignReleaseBean.java
package top.zbeboy.isy.web.bean.graduate.design.release;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.GraduationDesignRelease;

/**
 * Created by zbeboy on 2017/5/5.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class GraduationDesignReleaseBean extends GraduationDesignRelease {
    private String realName;
    private String departmentName;
    private String schoolName;
    private String collegeName;
    private String scienceName;
    private int schoolId;
    private int collegeId;
    private String fillTeacherStartTimeStr;
    private String fillTeacherEndTimeStr;
    private String startTimeStr;
    private String endTimeStr;
    private String releaseTimeStr;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/release/GraduationDesignReleaseBean.kt
package top.zbeboy.isy.web.bean.graduate.design.release

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignRelease

/**
 * Created by zbeboy 2018-01-14 .
 **/
class GraduationDesignReleaseBean: GraduationDesignRelease() {
    var realName: String? = null
    var departmentName: String? = null
    var schoolName: String? = null
    var collegeName: String? = null
    var scienceName: String? = null
    var schoolId: Int? = 0
    var collegeId: Int? = 0
    var fillTeacherStartTimeStr: String? = null
    var fillTeacherEndTimeStr: String? = null
    var startTimeStr: String? = null
    var endTimeStr: String? = null
    var releaseTimeStr: String? = null

    // 未填报学生数
    var studentNotFillCount: Int? = 0
    // 填报学生数
    var studentFillCount: Int? = 0
    // 同步时间
    var syncDate: String? = null
    var hasSyncDate: Boolean? = false
}